test/scale-infra/README.md

```diff
@@ -0,0 +1,69 @@
+A scalable Kubernetes-based testbed for KubeStellar Performance Tests
+---------------------------------------------------------------------
+
+<img src="images/ks-scale-test-infra.png" width="60%" height="60%" title="ks-scale-test-infra">
+
+### Requirements
+AWS EC2 is the environment used to setup the testbed. So AWS credentials are required.
+They can be set by environment variables, e.g. `AWS_ACCESS_KEY` and `AWS_SECRET_KEY`.
+
+### Overview
+
+The testbed consists of (from bottom to top):
+- Networking and stroage resources from AWS;
+- AWS EC2 instances;
+- Kubernetes cluster(s);
+- Workload(s): Kubestellar Core and WEC components 
+
+
+### Deployment Instructions
+
+1. Deploy the core region: 
```

Comment:
"core region" is an undefined term. It is not a term defined in KubeStellar in general, nor does it appear in the picture at the top of this page.

Reply:
The whole scale test-infra sucessfully works when everything is deployed into the same region. Agree, I removed the terms `core region` and `edge region`. I rename the scripts to the following new names: `deploy_wec_infra.sh` & `deploy_ks_cp_infra.sh`

Reply:
Good. This doc should mention somewhere up front that all the pieces of this system go in a single region.

Reply:
I added the following statement at step 2 in the README: 

```
Use the flag --wecs_hosting_instances to specify the number of ec2 instances to be created. You must deploy the WEC hosting instances in the same region as the KS control plane hosting infra created a step 1 - multiple regions deployment is not supported at the moment.
```